src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,47 @@
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 60px;
+    font-family: "Roboto", sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $color-main-accent;
+
+    &:hover {
+      cursor: pointer;
+      color: $color-accent-blue;
+      @include default-transition();
```

Comment:
don't add transition on hover, add it under general selector, then it will work both ways